content/php/concepts/math-functions/terms/octdec/octdec.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'octdec()'
+Description: 'Returns given octal numbers to decimal numbers.'
+Subjects:
+  - 'Web Development' 
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`octdec()`** function is a built-in function that calculates the decimal equivalent of an octal number. It accepts the octal number as the only argument.
+
+## Syntax
+
+```pseudo
+octdec(value);
+```
+
+- `value`: It's an octal number given as a parameter to the function in string format. If an invalid character is passed, it is ignored.
+
+## Example
+
+The example below uses the `octdec()` function to convert an octal number to its decimal value.
+
+```php
+<?php
+ echo octdec("65");
+ echo octdec("3098");
```

Comment:
```suggestion
echo octdec("65") . "<br>";
echo octdec("3098") . "<br>";
```
To print the output like below, you'll need `<br>` tags so the next statement will print on a new line.